ucp/_libs/ucx_api.pyx

```diff
@@ -386,6 +388,82 @@ cdef class UCXWorker(UCXObject):
         assert_ucs_status(status)
         return UCXEndpoint(self, <uintptr_t>ucp_ep)
 
+    cpdef ucs_status_t fence(self) except *:
+        cdef ucs_status_t status = ucp_worker_fence(self._handle)
+        assert_ucs_status(status)
+        return status
+
+    def flush(self, cb_func, cb_args, cb_kwargs=dict()):
+        cdef ucs_status_ptr_t req
+        cdef ucp_send_callback_t _send_cb = <ucp_send_callback_t>_send_callback
+
+        cdef ucs_status_ptr_t status = ucp_worker_flush_nb(self._handle, 0, _send_cb)
+        return _handle_status(
+            status, 0, cb_func, cb_args, cb_kwargs, 'flush', self._inflight_msgs
+        )
+
+    def get_address(self):
+        return UCXAddress(self)
+
+
+def _ucx_address_finalizer(uintptr_t handle_as_int, UCXWorker worker):
+    cdef ucp_address_t *address = <ucp_address_t *>handle_as_int
+    cdef ucp_worker_h ucp_worker = <ucp_worker_h><uintptr_t>worker.handle
+    ucp_worker_release_address(ucp_worker, address)
+
+
+cdef class UCXAddress(UCXObject):
+    """Python representation of ucp_address_t"""
+    cdef ucp_address_t *_handle
+    cdef size_t _length
+    cdef worker
+
+    def __init__(self, UCXWorker worker):
+        cdef ucs_status_t status
+        cdef ucp_worker_h ucp_worker = <ucp_worker_h><uintptr_t>worker.handle
+
+        status = ucp_worker_get_address(ucp_worker, &self._handle, &self._length)
+        assert_ucs_status(status)
```

Comment:
Would suggest just using a local variable that matches the `size_t` expected by UCX and then convert it to `Py_ssize_t` afterwards.

```suggestion
        cdef size_t length

        status = ucp_worker_get_address(ucp_worker, &self._handle, &length)
        assert_ucs_status(status)
        self._length = length
```

Reply:
This is where you get length, so here length will still be size_t but will be casted into Py_ssize_t when assigned to the object.